pccommon/setup.py

```diff
@@ -4,7 +4,7 @@
 
 # Runtime requirements.
 inst_reqs = [
-    "fastapi==0.67.*",
+    "fastapi==0.73.*",
```

Comment:
We had pinned this earlier due to OpenAPI docs page breaks - can you confirm those still render fine on local setup?

Reply:
They look correct at the top level, do you remember if the error was more nuanced?  An interesting note here was that, without this upgrade, I could not get pip to resolve dependencies with the upgraded `stac-fastapi.types` package, due to the [upgrade there](https://github.com/stac-utils/stac-fastapi/pull/337/commits/87385bf05d10f820249a147e3f45f3b0776a6923).

![image](https://user-images.githubusercontent.com/1014341/157924289-8ec429ec-9e0b-490e-aa45-bb7a96fb4dad.png)

Reply:
Nope, as I recall it wouldn't even render. LGTM